pkg/odo/cli/component/create.go

```diff
@@ -338,6 +342,10 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 			// Component type: Get from full command's first argument (mandatory)
```

Comment:
move the comments at the right place, above `componentType = args[0]`

Reply:
Yup, I can refactor the comments for both interactive mode and direct mode